package-lock.json

```diff
@@ -8172,19 +8172,6 @@
 				"url": "https://github.com/fb55/domutils?sponsor=1"
 			}
 		},
-		"node_modules/dotenv": {
```

Comment:
@ajjuaire Can make this file exactly as it was before and remove all changes here?